solutions/DataFormats.tests.oop/models/DataFormats.tests.oop.format.mps

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Tests for these concepts are missing:
* Alias
* Anchor
* Tag
* Timestamp

Please add tests for instances that contain single quotes and double quotes of:
* String
* Anchor
* Tag